hazelcast/assets/configuration/spec.yaml

```diff
@@ -33,6 +34,12 @@ files:
         port.display_priority: 9
         port.description: Hazelcast or Hazelcast Management Center port with which to connect.
         port.value.example: 1099
+    - template: instances/http
+      overrides:
+        password.name: http_password
+        password.description: |
+          The password to use if services are behind basic or NTLM auth.
+          Renamed to `http_password` from `password` to be compatible with JMX's config options.
```

Comment:
Actually, the http wrapper will still use the JMX `password` option, since `http_password` is not used anywhere, but I don't know if that's an issue

Reply:
It doesn't seem like `password` is used in the `hazelcast` check currently. I could exclude http's `password` option, and then enforce that the http password be the same as JMX password.

Reply:
AFAIU, the JMX password is implicitly used by the JMX check, so it does not appear in the code, like other JMX options.
After some search, I think nothing prevents the JMX password from being different from the HTTP one. The best solution may be to have `password` as the JMX option, and `http_password` as the HTTP password like you did. However, we need a logic in the Hazelcast code for the Hazelcast HTTP wrapper to use `http_password` instead of `password`.

@ofek What do you think?

Reply:
https://github.com/DataDog/integrations-core/pull/7968